ui/packages/shared/profile/src/Table/index.tsx

```diff
@@ -201,13 +201,13 @@ const CustomRowRenderer = ({
             {idx === 0 && isExpanded ? (
               <>
                 <div
-                  className={`absolute top-0 left-0 bg-white dark:bg-indigo-500 px-1 uppercase -rotate-90 origin-top-left z-10 text-[10px] border-l border-y border-gray-200 dark:border-gray-700 text-left`}
+                  className={`absolute top-0 left-0 bg-white dark:bg-indigo-500 px-1 uppercase -rotate-90 origin-top-left z-[9] text-[10px] border-l border-y border-gray-200 dark:border-gray-700 text-left`}
```

Comment:
This change fixes the z-index bug below.

<img width="554" alt="image" src="https://github.com/user-attachments/assets/f263148c-9f75-4237-8f01-d0606cb5599e">